api/py/ai/chronon/repo/run.py

```diff
@@ -237,6 +237,7 @@ def set_runtime_env(args):
         - Environment variables derived from args (like app_name)
         - conf.metaData.modeToEnvMap for the mode (set on config)
         - team environment per context and mode set on teams.json
```

Comment:
we should rename this to dev
```suggestion
        - team's dev environment for each mode set on teams.json    
```